### PR TITLE
qa/workunits: install nose before using it

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
```
2019-12-26T09:13:53.897 INFO:tasks.workunit.client.0.smithi096.stderr:+ dirname /home/ubuntu/cephtest/clone.client.0/qa/workunits/rbd/test_librbd_python.sh
2019-12-26T09:13:53.897 INFO:tasks.workunit.client.0.smithi096.stderr:+ relpath=/home/ubuntu/cephtest/clone.client.0/qa/workunits/rbd/../../../src/test/pybind
2019-12-26T09:13:53.898 INFO:tasks.workunit.client.0.smithi096.stderr:+ [ -n  ]
2019-12-26T09:13:53.898 INFO:tasks.workunit.client.0.smithi096.stderr:+ python3 -m nose -v /home/ubuntu/cephtest/clone.client.0/qa/workunits/rbd/../../../src/test/pybind/test_rbd.py
2019-12-26T09:13:53.919 INFO:tasks.workunit.client.0.smithi096.stderr:/usr/bin/python3: No module named nose
2019-12-26T09:13:53.923 DEBUG:teuthology.orchestra.run:got remote process result: 1
```

this change should address the failure of http://pulpito.ceph.com/kchai-2019-12-26_05:49:41-rados-wip-kefu-testing-2019-12-26-1055-distro-basic-smithi/4631733/

File: qa/workunits/rbd/test_librbd_python.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -ex
 
+source $(dirname $0)/../ceph-helpers-root.sh
+install python3-nose
```
Comment (dillaman) [inline qa/workunits/rbd/test_librbd_python.sh]: Isn't this handled here [1]?

[1] https://github.com/ceph/ceph-cm-ansible/blob/master/roles/testnode/vars/redhat_8.yml#L50

Comment (tchaikov) [reply qa/workunits/rbd/test_librbd_python.sh]: thanks, Jason. i am posting https://github.com/ceph/ceph-cm-ansible/pull/518 instead.

**Comment by tchaikov** [comment]
we need to install python3-nose on bionic test nodes as well as on RHEL8 nodes.

**Pull request closed without merging**